charts/argo-cd/README.md

```diff
@@ -76,6 +76,7 @@ Helm v3 has removed the `install-crds` hook so CRDs are now populated by files i
 | global.hostAliases | Mapping between IP and hostnames that will be injected as entries in the pod's hosts files | `[]` |
 | nameOverride | Provide a name in place of `argocd` | `"argocd"` |
 | installCRDs | Install CRDs if you are using Helm2. | `true` |
+| configs.clusterCredentials | [External Cluster Credentials](https://argoproj.github.io/argo-cd/operator-manual/declarative-setup/#clusters)| `[]` (See [values.yaml](values.yaml)) |
```

Comment:
Needs to be more precise. Something like "Provide one or multiple [external cluster credentials](https://argoproj.github.io/argo-cd/operator-manual/declarative-setup/#clusters)"